src/com/epam/javatrainings/annotations/Validator.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.javatrainings.annotations;
+
+import java.lang.annotation.AnnotationTypeMismatchException;
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+
+public class Validator {
+    public void validate(Dto obj)
+    {
+        try {
+            Field name=obj.getClass().getDeclaredField("name");
```

Comment:
what if we'll validate another DTO which does not have a field name? try to write more generic code, pass to validators an object, iterate over all fields and check if any field has a specific annotation, if so validate that field, if not just skip